shared/network/2.x.md

```diff
@@ -1,15 +1,18 @@
 # Networking on {{ $names.os.lower }} 2.x
 
-* [Introduction](#introduction)
-* [WiFi Setup](#wifi-setup)
-* [Regulatory Domain](#regulatory-domain)
-* [Setting a Static IP](#setting-a-static-ip)
-* [Creating a Hotspot](#creating-a-hotspot)
-* [Cellular Modem Setup](#cellular-modem-setup)
-* [Changing the Network at Runtime](#changing-the-network-at-runtime)
-* [Network Requirements](#network-requirements)
-* [Connecting Behind a Proxy](#connecting-behind-a-proxy)
-* [Checking Connectivity](#checking-connectivity)
+- [Networking on {{ $names.os.lower }} 2.x](#networking-on--namesoslower--2x)
+  - [Introduction](#introduction)
+  - [WiFi Setup](#wifi-setup)
+  - [Ethernet Setup](#ethernet-setup)
+  - [Regulatory Domain](#regulatory-domain)
+  - [Setting a Static IP](#setting-a-static-ip)
+  - [Creating a Hotspot](#creating-a-hotspot)
+  - [Cellular Modem Setup](#cellular-modem-setup)
+    - [Known Tested Modems](#known-tested-modems)
+  - [Changing the Network at Runtime](#changing-the-network-at-runtime)
+  - [Network Requirements](#network-requirements)
+  - [Connecting Behind a Proxy](#connecting-behind-a-proxy)
+  - [Checking connectivity](#checking-connectivity)
```

Comment:
Is your markdown lint extension making these changes to the contents or is it you?

Reply:
Yeah, but it was a bit overzealous so I toned it down on the last push.